### PR TITLE
refactor : UI refactor for cybersentinel page

### LINKED ISSUE #11: Will design the website with css and javascript
Redesign the structure of the website without changing any functionality

### PR DESCRIPTION
**Pull request by VineeTagarwaL-code**
fixes: #11

### CODE REVIEW

File: README.html [hunk 1]
```diff
@@ -20,117 +20,199 @@
...
+            <li>The vulnerability</li>
+            <li>How it can be exploited</li>
+            <li>The part of the code causing this vulnerability.</li>
+            <li>How to patch this vulnerability at different levels.</li>
+        </ul>
+
+        <h2>Installation</h2>
+        <ol>
+            <li>
+                To set up this application on Windows, you must run Apache and MySQL services, which are inbuilt in
```
Comment (SuchitReddi) [inline README.html:128]: The link for setupw.html is missing here

File: README.html [hunk 2]
```diff
@@ -20,117 +20,199 @@
...
+            <li>
+                After setting up and downloading the repository, rename the folder to cybersentinel and place it inside
+                htdocs
+            </li>
+            <li>
+                You can also download the dockerized version of the application here, which will require you to download
+                the Docker Desktop application for Windows.
+            </li>
+            <li>
+                Follow the steps in this Docker Hub repository to use this application on Docker.
```
Comment (SuchitReddi) [inline README.html:143]: The link for dockerhub is missing here.

File: README.html [hunk 3]
```diff
@@ -20,117 +20,199 @@
...
+
+
+        <h2>For developers</h2>
+        <ul>
+            <li>
+                Refer to CONTRIBUTING.md and DEVELOPMENT.md for more information on how to contribute to this project.
+            </li>
+        </ul>
+
+        <h2 style="text-align: center;">Disclaimer</h2>
```
Comment (SuchitReddi) [inline README.html:176]: From here onwards, text-align center is left unremoved.

File: README.html [hunk 4]
```diff
@@ -20,117 +20,199 @@
...
+        <p style="text-align: center;">This application is vulnerable! There are documented vulnerabilities that will
+            cause more undocumented ones.
+            So, it is strictly advised not to use it directly on your host machine without any virtual machine or
+            docker.
+            Do not host code from this project on internet-facing servers.</p>
+        <br />
+        <p style="text-align: center;">This application is vulnerable! There are documented vulnerabilities that will
```
Comment (SuchitReddi) [inline README.html:183]: "This application is vulnerable! There are documented vulnerabilities that will ".

This line is added unnecessarily in lines 183, and 191.

File: README.html [hunk 5]
```diff
@@ -20,117 +20,199 @@
...
+            </a>
+            <a href="https://discord.gg/sTwyENzBUb" target="_blank" rel="noopener noreferrer nofollow">
+                <button class="contact" style="width: 80px;">Discord</button>
+            </a>
+            <a href="https://join.slack.com/t/opensrccybersentinel/shared_invite/zt-28f973fci-7ObrO7JK8jE7FNG3qDiLvQ"
+                target="_blank" rel="noopener noreferrer nofollow">
+
+                <button class="contact" style="width: 80px;">Slack</button>
+            </a>
+
```
Comment (SuchitReddi) [inline README.html:211]: Credits is missing, and some content in contact is missing.

File: README.html (hunk 1 above)
Comment (VineeTagarwaL-code) [reply README.html:128]: Fixed

File: README.html (hunk 2 above)
Comment (VineeTagarwaL-code) [reply README.html:143]: fixed

File: README.html (hunk 3 above)
Comment (VineeTagarwaL-code) [reply README.html:176]: i did this intentionally  , if you didnt liked the idea we can change it back

File: README.html (hunk 4 above)
Comment (VineeTagarwaL-code) [reply README.html:183]: image
its there in your main branch already , look once

Comment (SuchitReddi) [reply README.html:187]: image
It is there only in the first paragraph, but in your file, it is present in the beginning of all paragraphs.

File: README.html (hunk 3 above)
Comment (SuchitReddi) [reply README.html:180]: It does look better this way. Let's leave it like this.

File: README.html (hunk 1 above)
Comment (SuchitReddi) [reply README.html]: Sorry, I gave the wrong line here. The link for setupw.html should be given in the next line "Follow this for setting up Cyber Sentinel on Windows."

Comment (VineeTagarwaL-code) [reply README.html]: fixed

**Pull request merged**